core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/utiltosqldate/UtilToSqlDateUtilsUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package com.baeldung.utiltosqldate;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.text.ParseException;
+
+public class UtilToSqlDateUtilsUnitTest {
+
+    @Test
+    public void given_utilDate_when_Casting_to_sqldate_throw_exception() {
+        Assertions.assertThrows(ClassCastException.class, () -> {
+            java.sql.Date date = (java.sql.Date) new java.util.Date();
+        });
+    }
+
+    @Test
+    public void given_utilDate_whenStandardConversion_timezone_lost() throws ParseException {
+        java.util.Date date = UnitToSqlDateUtils.createAmericanDate("2010-05-23T22:01:02");
+
+        java.sql.Date sqlDate;
+
+        UnitToSqlDateUtils.switchTimezone("America/Los_Angeles");
+        sqlDate = new java.sql.Date(date.getTime());
```

Comment:
It might make more sense to declare and set this value at once.

```suggestion

        UnitToSqlDateUtils.switchTimezone("America/Los_Angeles");

        java.sql.Date sqlDate = new java.sql.Date(date.getTime());
```